src/engine/bank01.asm

```diff
@@ -8477,7 +8478,11 @@ Func_74dc: ; 74dc (1:74dc)
 	inc de
 	jr .card_loop
 
-LinkDuel_WaitForStartButton: ; 7528 (1:7528)
+; seems to communicate with other device
+; for starting a duel
+; outputs in hl either wPlayerDuelVariables
+; or wOpponentDuelVariables depending on wSerialOp
+DecideLinkDuelVariables: ; 7528 (1:7528)
```

Comment:
I decided to maintain the function label I wrote (which in my opinion communicates the function of what is doing more specifically) but kept the local labels otherwise intact